app/api/sessions.py

```diff
@@ -54,8 +55,8 @@ def after_create_object(self, session, data, view_kwargs):
             organizer_email = organizer.email
             frontend_url = get_settings()['frontend_url']
             event = session.event
-            link = "{}/events/{}/sessions/{}"\
-                .format(frontend_url, event.identifier, session.id)
+            link = make_frontend_url("{}/events/{}/sessions/{}"
```

Comment:
Wrong use

Reply:
@iamareebjamal Sorry for that. Forgot to remove the FE url as a format string paramater. Fixed it now.